apps/hash-graph/README.md

```diff
@@ -38,25 +38,25 @@ If you're interested in just increasing the logs for the Graph, we recommend spe
 In order to build run the following command:
 
 ```shell
-cargo make build
+just build
 ```
 
 In order to create an optimized build, run:
 
 ```shell
-cargo make --profile production build
+just --profile production build
```

Comment:
Does this work? It looked like the `.justfile` requires it to come from the env?

Reply:
Cheers, can you have a quick scan to see if there are any other similar ones. Maybe in `dockerfile`s or something